src/app/clothes/page.tsx

```diff
@@ -0,0 +1,87 @@
+'use client';
+import { useState, useEffect } from 'react';
+import { Button } from '@/components/ButtonStyle';
+import { getCategory } from '@/functions/getCategory';
+import { getData } from '@/functions/getData';
+import { normalizeImage } from '@/functions/normalizeImage';
+import { stringifyUrl } from '@/functions/stringifyUrl';
+import { ItemClothes } from '@/types/itemClothes';
+import { Card, CardFooter, Image } from '@nextui-org/react';
+import Link from 'next/link';
+import React from 'react';
+import { dataClothes } from '@/mocks';
+
+const Page = () => {
+  const [data, setData] = useState<ItemClothes[]>([]);
+  const [selectedCategory, setSelectedCategory] = useState<string>('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const fetchedData: ItemClothes[] = await getData();
+      setData(fetchedData);
+    };
+
+    fetchData();
+  }, []);
```

Comment:
@samuelrms prefer isolating your fetch effect and state into a custom hook instead, to avoid putting addtl layout logic into your page. Even better, for this case, SSR would've worked wonders: https://nextjs.org/docs/pages/building-your-application/rendering/server-side-rendering

Reply:
While building a few projects ago, I realized that the use of getServerSideProps, getStaticProps, getStaticPaths and others, are no longer available in version 13.4 using AppDir as the documentation itself indicates, so I am using this function within my page, this issue of building a hook is super interesting, I will be implementing it!

The link where you find the change I mentioned is this one 
https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#async-and-await-in-server-components

Reply:
@samuelrms cool! glad you've found yourself on it =)